mmaction/core/evaluation/accuracy.py

```diff
@@ -104,17 +104,46 @@ def top_k_accuracy(scores, labels, topk=(1, )):
     return res
 
 
+def mmit_mean_average_precision(scores, labels):
```

Comment:
does this metric have a more `neutral` name?

Reply:
Might be hard, since I only find that the MMIT challenge uses this definition for mean average precision. All the others use the normal definition like detection.